src/sentry/api/endpoints/seer_rpc.py

```diff
@@ -461,6 +466,44 @@ def get_attributes_and_values(
     return {"attributes_and_values": attributes_and_values}
 
 
+def get_github_enterprise_integration_config(
+    *, organization_id: int, integration_id: int
+) -> dict[str, Any]:
+    if not settings.SEER_GHE_ENCRYPT_KEY:
+        raise RuntimeError("Cannot encrypt access token without SEER_GHE_ENCRYPT_KEY")
+
+    integration = integration_service.get_integration(
+        integration_id=integration_id,
+        provider=IntegrationProviderSlug.GITHUB_ENTERPRISE.value,
+        organization_id=organization_id,
+        status=ObjectStatus.ACTIVE,
+    )
+    if integration is None:
+        raise Integration.DoesNotExist
+
+    installation = integration.get_installation(organization_id=organization_id)
+    assert isinstance(installation, GitHubEnterpriseIntegration)
+
+    client = installation.get_client()
+    access_token = client.get_access_token()
+
+    if not access_token:
+        raise RuntimeError("No access token found")
+
+    try:
+        fernet = Fernet(settings.SEER_GHE_ENCRYPT_KEY.encode("utf-8"))
+        encrypted_access_token = fernet.encrypt(access_token.encode("utf-8")).decode("utf-8")
+    except Exception:
+        sentry_sdk.capture_exception()
```

Comment:
Something to double-check, but are we configured to capture local variables? We don't want to accidentally leak the `access_token` value.

Maybe a simple fix would be to call `del access_token` before capturing? 🤷‍♂️ 

Otherwise, everything else looks good.

Reply:
@mdtro Another option is to log the error and return `None` instead of raising an exception (and let the SeerRpcServiceEndpoint wrapper handle the response). What do you think?